src/hotspot/share/gc/parallel/psParallelCompact.hpp

```diff
@@ -190,9 +190,7 @@ class SpaceInfo
   void set_dense_prefix(HeapWord* addr)     { _dense_prefix = addr; }
   void set_start_array(ObjectStartArray* s) { _start_array = s; }
 
-  void publish_new_top() const              { _space->set_top(_new_top); }
-
- private:
+private:
```

Comment:
Given that in this class (`SpaceInfo`) the existing spacing for visibility specifiers uses a space in front I would keep it that way (or change all of them). Mixing them within a single class seems just too strange, but it's your call in the end.

Reply:
Removed all preceding space.